apps/backend/src/notification/entities/alert-destination.entity.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  Column,
+  CreateDateColumn,
+  PrimaryGeneratedColumn,
+  Entity,
+  ManyToOne,
+  JoinColumn,
+} from 'typeorm';
+import { FileIngestionRulesEntity } from '../../parse/entities/file-ingestion-rules.entity';
+import { ProgramRequiredFileEntity } from '../../parse/entities/program-required-file.entity';
+
+@Entity('alert_destination')
+export class AlertDestinationEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @CreateDateColumn({ name: 'created_date' })
+  createdDate: Date;
+
+  /**
+   * Only one of the next three must be truthy
+   */
```

Comment:
should 'allAlerts' and/or 'requiredFile' be optional? (ie: 'rule?')

Reply:
Technically we don't have validation to ensure that only one of the three is truthy, but yeah they should all be optional. Happy to get on a call to discuss